### ISSUE TITLE
`collect` has no method matching collect(::Tuple{DataType,DataType}, ::Hexagons.HexagonVertexIterator)

### ISSUE THREAD

**Issue by GordStephen** [dcjones/Hexagons.jl#4]
`Geom.hexbin` plotting with Gadfly returns an error that traces back to [`hexpoints`](https://github.com/dcjones/Hexagons.jl/blob/master/src/Hexagons.jl#L194) - there's no collect method defined for use with a `HexagonVertexIterator`.

``` julia
ERROR: MethodError: `collect` has no method matching collect(::Tuple{DataType,DataType}, ::Hexagons.HexagonVertexIterator)
```

**Comment by timholy** [comment]
Already fixed by @dcjones in 1ce465e9dedff12efc1bb5c36292bb6f54ff6d41, just needs a new release to be tagged. Meanwhile, you can checkout the master branch.

**Comment by tlnagy** [comment]
Fixed in v0.0.4
